ene_008_oil_chemical_spill_incidents/Dockerfile

```diff
@@ -1,18 +1,17 @@
-FROM python:3.6-alpine
-MAINTAINER Amelia Snyder <amelia.snyder@wri.org>
-#Note this script was originally developed by Nathan Suberi <nathan.suberi@wri.org>
+FROM python:3.9
+MAINTAINER Weiqi Zhou <weiqi.zhou@wri.org>
+#Note this script was originally developed by Nathan Suberi <nathan.suberi@wri.org> and maintained by Amelia Snyder
 
 # install core libraries
-RUN apk update
-RUN apk add git
-RUN pip install --upgrade pip
+RUN apt-get update
+RUN pip install -U pip
 
 # install application libraries
-RUN pip install requests==2.22.0
+RUN pip install requests
 RUN pip install -e git+https://github.com/resource-watch/cartosql.py.git#egg=cartosql
 
 # env
-ARG NAME=ene_008
+ARG NAME=nrt-script
```

Comment:
Seems like some use `nrt-script` and others use the dataset id. Looks like this is just used as the name of the directory in Docker so seems fine to standardize on `nrt-script`.

Reply:
Updated in PR https://github.com/resource-watch/nrt-scripts/pull/1182